pkg/controllers/machine_controller.go

```diff
@@ -235,6 +244,33 @@ func (r *MachineReconciler) startCheckAndEnqueueVolumeResize(ctx context.Context
 	}, r.resyncIntervalVolumeSize)
 }
 
+func (r *MachineReconciler) startCheckAndEnqueueMachineState(ctx context.Context, log logr.Logger) {
+	wait.UntilWithContext(ctx, func(ctx context.Context) {
+		machines, err := r.machines.List(ctx)
+		if err != nil {
+			log.Error(err, "failed to list machines")
+			return
+		}
+
+		for _, machine := range machines {
+			if machine.DeletedAt != nil || !slices.Contains(machine.Finalizers, MachineFinalizer) {
+				continue
+			}
+			id := machine.ID
+
+			state, err := r.getMachineState(id)
+			if err != nil {
+				log.Error(err, "failed to get machine state", "machineID", id)
```

Comment:
```suggestion
				log.Error(err, "failed to get machine state", "machineId", id)
```

Pls keep same keys between logs. Filtering can be case sensitive

Reply:
@lukas016 Don't you think it's better to be consistent? Across the entire project we use `machineID` (for logging and as var) why should we change it here?

Reply:
no problem, but pls create issue for unified https://github.com/ironcore-dev/libvirt-provider/blob/main/pkg/controllers/machine_controller.go#L246

Reply:
just this one is different. So we can fix it in this PR and we don't need issue. What do you think @lukasfrank

Reply:
& here  https://github.com/ironcore-dev/libvirt-provider/blob/main/pkg/controllers/machine_controller.go#L211-L229
and the entire variable naming

Reply:
Following our offline discussion, I've modified the key [here](https://github.com/ironcore-dev/libvirt-provider/blob/864eba2245b1606aa06f2a684e624bb32a33a6c8/pkg/controllers/machine_controller.go#L246) to `machineID`.